js/js-array-methods/js-array-methods_map/index.js

```diff
@@ -24,10 +24,18 @@ const cards = [
   },
 ];
 
-const lowerCaseAnswers = null; // ['as often as you like.', ...]
+const lowerCaseAnswers = cards.map((element) => {
+  return element.answer.toLocaleLowerCase();
```

Comment:
I wrote "return element.answer.toLowerCase();" but if it works for you like that ignore my comment :)